assignment3.txt

```diff
@@ -0,0 +1,7 @@
+1. The code is not valid as the syntax for the for loop is incorrect. First, i is never declared before it is used and is not given a starting value. Inside the loop, there is also no integer in the printf statement. 
+
+2. The variable i should not be reused within the loop because the computer will not be able to differentiate between the i being used to increment in the loop and the i in loop.
+
+3. i. O(2n) -- worst case scenario it would be O(n) for each loop --> O(2n)
```

Comment:
Big O notation doesn't care about constants or lower terms, so the 2 would go away regardless. But also, since it's a nested loop, it would n * n = n^2.